examples/federated-learning/tff/nvflare/base-client/base-client.yaml

```diff
@@ -0,0 +1,79 @@
+---
```

Comment:
There's  some more duplication to reduce between this file and `base-server-deployment.yaml` because there are only minor differences between the two.